src/api/APICalls.jsx

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+
+const articleAPI = axios.create({
+  baseURL: "https://lizzie-hughes-nc-news-project.onrender.com/api",
+});
+
+export const getAllArticles = () => {
+  return articleAPI
+    .get(`/articles`)
+    .then((response) => {
+      return response.data.articlesWithComments;
+    })
+    .catch((err) => console.log(err));
+};
+export const getSingleArticle = (article_id) => {
+  return articleAPI
+    .get(`/articles/${article_id}`)
+    .then((response) => {
+      return response.data.selectedArticle;
+    })
+    .catch((err) => console.log(err));
+};
+export const getCommentByArticleID = (article_id) => {
+  return articleAPI
+    .get(`/articles/${article_id}/comments`)
+    .then((response) => {
+      return response.data.comments;
+    })
+    .catch((err) => console.log(err));
```

Comment:
The .catch(...) methods should not be here for any of these. We should allow this to fail so it can enter the .catch on the component side instead